aii-ks/src/main/pan/quattor/aii/ks/schema.pan

```diff
@@ -57,7 +57,7 @@ type structure_ks_logging = {
 type structure_ks_ks_info = {
     "ackurl"	: type_absoluteURI
     "acklist"	? type_absoluteURI[]
-    "auth"		: string[] = list ("enableshadow", "enablemd5")
+    "auth"		: string[] = list ("enableshadow", "passalgo=sha512")
```

Comment:
yes, just this line is enough. if you do larger search and replace, make sure you do it all in one separate whitespace-only commit